code/modules/mob/living/carbon/human/human.dm

```diff
@@ -50,6 +50,7 @@
 
 	internal_organs += new dna.species.mutanteyes()
 	internal_organs += new /obj/item/organ/brain
+	give_genitals()
```

Comment:
I already added a check in give_genitals now though? It's not... like... better to check outside it. The proc responsible for handling genitals should be doing the checks, right?